docs/src/components/mdx/code-block.tsx

```diff
@@ -0,0 +1,127 @@
+import Button from '@pluralsight/ps-design-system-button'
+import Theme, { useTheme } from '@pluralsight/ps-design-system-theme'
+import cx from 'classnames'
+import Prism from 'prismjs/components/prism-core'
+import Highlight, { PrismTheme, defaultProps } from 'prism-react-renderer'
+import React, { HTMLAttributes, useEffect, useState } from 'react'
+import { CopyToClipboard } from 'react-copy-to-clipboard'
+
+import styles from './code-block.module.css'
+import { darkTheme, lightTheme } from './code-block-theme'
+
+interface CodeBlockProps extends HTMLAttributes<HTMLDivElement> {
+  live?: boolean
```

Comment:
this isn't used yet...but i'm expecting that it will be to render a preview

https://mdxjs.com/guides/live-code